test/Interpreter/EvalSpec.hs

```diff
@@ -41,9 +41,54 @@ spec = do
     testProgramm "1 < 3" (Bool True)
     testProgramm "3 <= 3" (Bool True)
     testProgramm "3 == 2" (Bool False)
+    testProgramm "9 /= 10" (Bool True)
+    testProgramm "4 /= 6" (Bool True)
+    testProgramm "True /= 1" (Bool False)
+    testProgramm "True == 1" (Bool True)
+    testProgramm "True /= 0" (Bool True)
+    testProgramm "0 /= 1" (Bool True)
+    testProgramm "5 /= 5" (Bool False) 
 
-  describe "more complex cases" $ do
+
+  describe "More complex cases" $ do
     -- basic variable usage
+
+
+    let condA = [r|
+      if 4 /= 3 then   
+        True    
+      end
+    |]
+
+    testProgramm condA (Bool True)
+
+
+    let condH = [r|
+      if 3 /= 3 then   
+        False    
+      end
+    |]
+
+    testProgramm condH (Empty)
+
+
+    let condZ = [r|
+      if 3 /= 4 then   
+        True    
+      end
+    |]
+
+    testProgramm condZ (Bool True)
+
+
+    let condX = [r|
+      if 0 /= 2 then
+        True
+      end
+    |]
```

Comment:
It's enough to  test the `/=` operator alone, `if` conditions are already covered